.github/workflows/count.yaml

```diff
@@ -0,0 +1,28 @@
+name: Count tabs number in changed files
```

Comment:
This file should not be here, as functional part current repo, please move to your report folder

Reply:
I still see this file, please remove 
![image](https://user-images.githubusercontent.com/8258487/205277420-53cad488-33ca-46d5-ae42-addbf6e7dca7.png)

Reply:
Already deleted it